README.md

```diff
@@ -1 +1,2 @@
-# hello-microverse
\ No newline at end of file
+# hello-microverse
+A new project to increatse our knowledge about Github , Github Flow and Pull requests.
```

Comment:
Your README file is not descriptive enough. Please add some more details.

Reply:
Hi. Thank you for the review. I added the changes. could you look again?